src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -50,6 +56,27 @@ class HaConfigHardware extends LitElement {
   }
 
   protected render(): TemplateResult {
+    let board: string | undefined;
+    let hostName: string | undefined;
+    let imageURL: string | undefined;
+    let documentationURL: string | undefined;
+
+    if (
+      isComponentLoaded(this.hass, "hardware") &&
```

Comment:
No need for this check, `this._hardwareInfo` will be undefined if there is no hardware integration